topics/epigenetics/tutorials/cut_and_run/workflows/main_workflow.ga

```diff
@@ -52,48 +57,69 @@
             "inputs": [
                 {
                     "description": "",
-                    "name": "GATA1_ChIP-Seq peaks"
+                    "name": "GATA1 ChIP-Seq peaks"
                 }
             ],
-            "label": "GATA1_ChIP-Seq peaks",
+            "label": "GATA1 ChIP-Seq peaks",
             "name": "Input dataset",
             "outputs": [],
             "position": {
-                "left": 126.3413499631912,
-                "top": 845.5851267270034
+                "left": 2450,
+                "top": 590
             },
             "tool_id": null,
             "tool_state": "{\"optional\": false, \"tag\": null}",
             "tool_version": null,
             "type": "data_input",
-            "uuid": "21729f16-ea35-477f-ad9b-e32e93275d36",
+            "uuid": "b920b82f-4ea8-4c40-8612-569abf7085da",
             "when": null,
             "workflow_outputs": []
         },
         "2": {
-            "annotation": "I certify that I am not using this tool for commercial purposes.",
-            "content_id": null,
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/trim_galore/trim_galore/0.6.7+galaxy1",
```

Comment:
I think the input "I certify that I am not using this tool for commercial purposes" must stay an input of the workflow, if not it will be lost.

Reply:
Yes